keps/sig-api-machinery/4988-snapshottable-api-server-cache/README.md

```diff
@@ -296,7 +344,8 @@ additional value on top of feature tests themselves.
 
 ###### What specific metrics should inform a rollback?
 
-Mismatch in hash label for different storage exposed by `apiserver_storage_hash` metric by the same apiserver.
+Snapshotting should automatically fallback to serving from etcd if inconsistency is detected.
+Rollback should be consider if there is a high number of inconsistencies detected by `storage_consistency_checks_total` metric.
 
 ###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
```

Comment:
answer missing and the rollout fail above.  Should be straightforward answers since this is read-only with automated fallback, but still needs to be answered.

Reply:
Done